home/screens/ProjectsScreen.js

```diff
@@ -327,6 +327,12 @@ export default class ProjectsScreen extends React.Component {
         <Text style={styles.expoVersionText} onPress={this._copyClientVersionToClipboard}>
           Client version: {Constants.expoVersion}
         </Text>
+        <Text style={styles.supportSdksText}>
+          Supports SDK{Constants.supportedExpoSdks.length === 1 ? "" : "s"} {Constants.supportedExpoSdks.sort().map(
```

Comment:
let's make this easier to read by doing the following to `Constants.supportedExpoSdks`:
- call `sort()`: we now have an array like `[31.0.0, 32.0.0, 33.0.0]`
- call `map()`, but use the `semver` pkg to get the major version. `substr` works in this case, but it is not as robust as using `semver`, which will be correct for all valid semvers.  (we now have an array like `[31, 32, 33]`)
- call `join()`, using `,` as the delimiter parameter. (we now have a string like `31, 32, 33`)

Reply:
fyi - be careful with sort since it uses alphabetical comparison by default (`['8.0.0', '9.0.0', '10.0.0'].sort()` -> `['10.0.0', '8.0.0'. '9.0.0']`.

So either we need to provide a custom comparator function, or do the `map` step first: `array.map(version => semver.major(version)).sort()`

Reply:
Thanks for the suggestions! Updated. :)